roles/nginxplus/files/conf/http/figgy-staging.conf

```diff
@@ -50,7 +50,7 @@ server {
         proxy_cache_convert_head off;
         proxy_cache_methods GET HEAD;
         proxy_cache_key $scheme$request_method$proxy_host$request_uri;
-        health_check interval=10 fails=3 passes=2;
+        health_check uri=health.json interval=10 fails=3 passes=2;
```

Comment:
Does it matter that this one doesn't have the slash in front of `health.json`?

Reply:
Good catch, fixed.